assets/dev/js/editor/components/template-library/component.js

```diff
@@ -161,6 +161,11 @@ export default class Component extends ComponentModalBase {
 				model: callbackParams.model,
 				data,
 				options: callbackParams.importOptions,
+				onAfter: () => {
+					this.manager.eventManager.sendTemplateInsertedEvent( {
+						library_type: callbackParams.model.get( 'source' ),
```

Comment:
```suggestion
						library_type: callbackParams.model.get( 'source' ) ?? 'local',
```
Not sure why I get a feeling we need to add a fallback here.